tests/dns-lua-rules/test-rrname.lua

```diff
@@ -1,11 +1,14 @@
+local dns = require("suricata.dns")
+
 function init (args)
    local needs = {}
-   needs["dns.rrname"] = tostring(true)
+   needs["dns.request"] = true
    return needs
 end
```

Comment:
This could also be:
```
return {["dns.request"] = true}
```
The `[]` is a way of quoting the string such that `.` can be used.